src/Pages/Register.js

```diff
@@ -1,61 +1,83 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom'
-import axios from 'axios'
-
+// import { Link } from 'react-router-dom'
+import axios from 'axios';
 
 function Registration() {
```

Comment:
I would suggest naming the component the same as the name of the file.   It's just easier to keep track of everything if you can always assume you're exporting Register from Register.js without having to go in and look